utbot-core/src/main/kotlin/org/utbot/common/ClassLoaderUtil.kt

```diff
@@ -0,0 +1,11 @@
+package org.utbot.common
+
+import java.net.URLClassLoader
+
+/**
+ * Checks that the class given by its binary name can be loaded with given classLoader.
+ */
+fun URLClassLoader.canLoad(classBinaryName: String): Boolean {
```

Comment:
```suggestion
fun URLClassLoader.isOnClasspath(classBinaryName: String): Boolean {
```
Also, the current version of documentation fails to convey that this function will return `true` despite `loadClass` throwing `NoClassDefFoundError` when the class is on class path, while it's parent or some interfaces are not.

Reply:
hasOnClasspath